changelog/add-7711-instant-deposit-currency

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: update
+
+Pass currency parameter and not transaction_ids parameter when creating instant deposit.
```

Comment:
Should we add a changelog entry for this kind of change, one that is not user facing and a change that is only 'in the background'?

I assume yes but like a confirmation.